### DISCUSSION TITLE
Are docker named volumes persisted?

### DISCUSSION THREAD

**Discussion by whollacsek** [Dokploy/dokploy#704] [Q&A] [answered]
In my docker compose config I have the following volumes section, however in the advanced page I don't see them being detected. Does this mean those volumes are not persisted?

```docker-compose.yml
volumes:
  docker-data:
  db-data:
  server-local-data:
```
It would be great if this behaviour could be documented so we know if we can rely on copy-pasting docker-compose.yml files as is.

**Comment by Siumauricio** [comment, marked as answer]
we don't parse or modify your docker compose that's why we don't show the volumes, so you can copy and paste any docker compose and it should work fine.

the only thing we offer is file mounts(volumes) that are useful for certain applications like nginx that need configuration files.

**Reply by whollacsek** [reply]
Thank you for the clarification
